client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -49,6 +49,14 @@ function getActiveLegendItems(
         legendItems.add("no");
         continue;
       }
+      const firstSupportItem = getFirst(browserSupport);
+      if (
+        firstSupportItem.notes &&
+        !firstSupportItem.partial_implementation &&
+        !firstSupportItem.version_removed
+      ) {
```

Comment:
Then you could reuse this here, and we no longer have to worry about changing one and forgetting the other:

```suggestion
      if (hasNoteworthyNotes(firstSupportItem.notes)) {
```